Bottom: f94d043d236a493fc86bdf7cd4b25cfa3d060d8b
Top:    ec87c1e1501417e96f30e7b89b9ae090bba78d7a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 11:12:25 -0500

Create new role nexus to configure Nexus3 service


---
diff --git a/ansible-playbook/group_vars/all/credentials.yml b/ansible-playbook/group_vars/all/credentials.yml
index c306cadf..88c2fbdc 100644
--- a/ansible-playbook/group_vars/all/credentials.yml
+++ b/ansible-playbook/group_vars/all/credentials.yml
@@ -11,6 +11,9 @@ gitlab_secrets_db_key_base:          '{{ lookup("password", playbook_dir + "/cre
 mantisbt_db_password:                '{{ lookup("password", playbook_dir + "/credentials/mantis/db/user"                      , length=20) }}'
 mantisbt_db_root_password:           '{{ lookup("password", playbook_dir + "/credentials/mantis/db/root"                      , length=20) }}'
 mantisbt_email_password:             '{{ lookup("password", playbook_dir + "/credentials/mantis/email/mantis@novatech-llc.com", length=20) }}'
+nexus_admin_password:                '{{ lookup("password", playbook_dir + "/credentials/nexus/users/admin"                   , length=20) }}'
+nexus_buildbot_password:             '{{ lookup("password", playbook_dir + "/credentials/nexus/users/buildbot"                , length=20) }}'
+nexus_email_password:                '{{ lookup("password", playbook_dir + "/credentials/exim4/nexus@novatech-llc.com"        , length=20) }}'
 openldap_admin_password:             '{{ lookup("password", playbook_dir + "/credentials/openldap/cn=admin,dc=novatech"       , length=20) }}'
 openldap_proxyagent_password:        '{{ lookup("password", playbook_dir + "/credentials/openldap/cn=proxyagent,dc=novatech"  , length=20) }}'
 svn_password:                        '{{ lookup("password", playbook_dir + "/credentials/svn/ansible"                         , length=20) }}'
diff --git a/ansible-playbook/group_vars/docker-hosts/networking.yml b/ansible-playbook/group_vars/docker-hosts/networking.yml
index f908c8f0..e94bb8ac 100644
--- a/ansible-playbook/group_vars/docker-hosts/networking.yml
+++ b/ansible-playbook/group_vars/docker-hosts/networking.yml
@@ -47,3 +47,8 @@ phpldapadmin_port_args:
 - "{{ phpldapadmin_ip_addr }}:443:443"
 
 phpmyadmin_hostname: phpmyadmin.novatech-llc.com
+
+nexus_hostname: nexus.novatech-llc.com
+nexus_port_args:
+- "{{ nexus_ip_addr }}:80:8081"
+- "{{ nexus_ip_addr }}:443:8443"
diff --git a/ansible-playbook/host_vars/daedalus.novatech-llc.com/networking.yml b/ansible-playbook/host_vars/daedalus.novatech-llc.com/networking.yml
index c6323198..2d8c4735 100644
--- a/ansible-playbook/host_vars/daedalus.novatech-llc.com/networking.yml
+++ b/ansible-playbook/host_vars/daedalus.novatech-llc.com/networking.yml
@@ -15,3 +15,4 @@ exim4_ip_addr:        172.16.0.107
 openldap_ip_addr:     172.16.0.108
 phpldapadmin_ip_addr: 172.16.0.108
 phpmyadmin_ip_addr:   172.16.0.109
+nexus_ip_addr:        172.16.0.111
diff --git a/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/networking.yml b/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/networking.yml
index b3e366ce..8d8fabe4 100644
--- a/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/networking.yml
+++ b/ansible-playbook/host_vars/testdaedalus.novatech-llc.com/networking.yml
@@ -15,3 +15,4 @@ exim4_ip_addr:        172.16.103.107
 openldap_ip_addr:     172.16.103.108
 phpldapadmin_ip_addr: 172.16.103.108
 phpmyadmin_ip_addr:   172.16.103.109
+nexus_ip_addr:        172.16.103.111
